3drepo/3drepo.io
# Issue 3774 - Custom Ticket comments endpoints

This fixes #3774

#### Description

 The following endpoints have been added:
- Get Comment
- Get Comments
- Create Comment
- Update Comment
- Delete Comment

#### Test cases

- Get Comment: The user should be able to retrieve comment details.
- Get Comments: The user should be able to retrieve a ticket's comment list.
- Create Comment: The user should be able to create a new comment.
- Update Comment: The user should be able to update an existing comment.
- Delete Comment: The user should be able to delete a comment.
